Add ability to turn off redshifting masses in TOV transforms

cdcapano opened the pull request:
This adds an option to the `LambdaFromTOVFile` transform to turn off redshifting masses. This is needed if you want to use this transform with a prior that is defined in the source frame (since the masses are already redshifted). The default is to redshift, as before. To turn it off, you add `do-not-redshift-mass =` to the transform's section in the config file.

This also will check if the requested mass is larger than the maximum mass supported by the equation of state. If so, it will set lambda to 0, assuming that it is a black hole.

cdcapano commented:
@soumide1102 poke

cdcapano commented:
@soumide1102 pokity poke

soumide1102 reviewed (dismissed):
@cdcapano Sorry, I was traveling and missed this.

One request---It might be nice to add something to the docstring for `LambdaFromTOVFile` to describe this as well. Otherwise looks fine to me. 

Looks like Travis is failing while processing dependencies etc, not because of anything related to the changes in this PR. Code Climate's complaints can be ignored.

ahnitz commented:
@cdcapano @soumide1102 What's the status of this?

cdcapano commented:
@ahnitz I added the doc strings as @soumide1102 requested, and rebased to current master. I'm not sure Soumi is checking PyCBC much anymore, so I've added @ahnitz as a reviewer.

cdcapano commented:
Hmm... looks like I can't merge until @soumide1102 signs off.
